src/alexa_go_vote/core.cljs

```diff
@@ -25,10 +41,8 @@
         intent (get-in request [:intent :name])]
    (cond
      (= type "LaunchRequest") launch-response
-     (= intent "pollingPlace") (pp/intent session request)
-     ;;check, I think there may be a new intent for didn't understand
-     :else (default-response event)
-     )))
-
-     ;; You can replace these response maps with calls to other functions that return similar maps.  The maps should correspond to the JSON requirements set forth
-     ;; here:   https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/alexa-skills-kit-interface-reference#response-format
+     (= intent "pollingPlace") (pp/intent request)
```

Comment:
Depending on how much you expect this to change going forward, this may be a good candidate to structure as a multimethod vs. `cond`(/`case`/`condp`/etc.).

Reply:
I see your point. I'll put a note in to do that if/when we implement other related queries (like, "where can I drop off my ballot" or "where can I vote early").